### DISCUSSION TITLE
title static-pages Breadcrumb and dynamic pages

### DISCUSSION THREAD

**Discussion by oim37** [wintercms/winter#871] [Q&A] [answered]
Hello,

In template i use {{ this.page.title}} and it works good in all pages. 90% pages used in static-pages module.
If i change {{ this.page.title}} to {% component 'staticBreadcrumbs' %} title working with breadcrumbs, i add styles and all look good. 

But if i go to dynamic page title is not displayed and it is logical, how to make a condition if the module static-pages used 
this active {% component 'staticBreadcrumbs' %}
else {{ this.page.title}}

Thank you.

**Comment by bennothommo** [comment, marked as answer]
If you have the `staticPage` component in the layout, you can use the following condition to detect if a static page is being viewed:

```twig
{% if staticPage.pageObject %}
    Static page loaded
{% else %}
    Dynamic page loaded
{% endif %}
```

The `$pageObject` property of the `staticPage` component will be `null` if no static page could be found for the following URL, or if you are using a normal CMS page for that URL.

**Reply by oim37** [reply]
That's what I need. Thank you very much!
